deploy/skaffold/Dockerfile

```diff
@@ -14,35 +14,35 @@
 
 # Download kubectl
 FROM alpine:3.10 as download-kubectl
-ENV KUBECTL_VERSION v1.12.8
+ENV KUBECTL_VERSION v1.17.1
 ENV KUBECTL_URL https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl
 RUN wget -O kubectl "${KUBECTL_URL}"
 RUN chmod +x kubectl
 
 # Download helm
 FROM alpine:3.10 as download-helm
-ENV HELM_VERSION v2.12.0
+ENV HELM_VERSION v2.16.1
```

Comment:
Our integration tests are not prepared for this change yet. Could you not update helm in this PR until we fix our test infrastructure?

Reply:
Sure thing.